app/views/wishlists/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="container">
   <h2>
     <%= @wishlist.name %>
-    <%= social_share_button_tag("Are you a hero of play? #{@wishlist.name} needs your help!") %>
+    <%= social_share_button_tag("Are you a hero of play? #{@wishlist.name} needs your help!",:url => "https://project-playtime-staging.herokuapp.com/wishlists/#{@wishlist.id}",desc: "Are you a hero of play? #{@wishlist.name} needs your help!") %>
```

Comment:
This URL changes. Is there a way to do this without a url attribute? If not, can we get this using environment variables?